codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -170,12 +170,15 @@ pub enum Network {
     #[subenum(HypersyncNetwork, NetworkWithExplorer)]
     GnosisChiado = 10200,
 
-    #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
+    #[subenum(NetworkWithExplorer, GraphNetwork)]
     Goerli = 5,
 
     #[subenum(HypersyncNetwork, NetworkWithExplorer)]
     Harmony = 1666600000,
 
+    #[subenum(HypersyncNetwork)]
+    Hyperliquid = 645749,
```

Comment:
:pray: 

Thank you - hyperliquid is currently not completely working - but very excited to have it in this release in anticipation!